rollout/trafficrouting/istio/istio_types.go

```diff
@@ -30,23 +48,23 @@ type VirtualServiceHTTPRouteDestination struct {
 
 // Destination fields within the VirtualServiceDestination struct of the Virtual Service that the controller modifies
 type VirtualServiceDestination struct {
-	Host   string `json:"host,omitempty"`
+	Host string   `json:"host,omitempty"`
 	Subset string `json:"subset,omitempty"`
 }
 
 // DestinationRule is an Istio DestinationRule containing only the fields which we care about
 type DestinationRule struct {
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              DestinationRuleSpec `json:"spec,omitempty"`
+	metav1.ObjectMeta        `json:"metadata,omitempty"`
+	Spec DestinationRuleSpec `json:"spec,omitempty"`
 }
 
 type DestinationRuleSpec struct {
 	Subsets []Subset `json:"subsets,omitempty"`
 }
 
 type Subset struct {
-	Name   string            `json:"name,omitempty"`
-	Labels map[string]string `json:"labels,omitempty"`
+	Name string                  `json:"name,omitempty"`
```

Comment:
go fmt?

Reply:
Sorry I forgot to run `go fmt` before pushing the changes. Will run and update the commit. Thanks!